src/cljs/org/broadinstitute/firecloud_ui/paths.cljs

```diff
@@ -45,5 +45,8 @@
 (defn submission-details [workspace-id submission-id]
   (str (submissions-list workspace-id) "/" submission-id))
 
+(defn submission-kill [workspace-id submission-id]
+  (submission-details workspace-id submission-id))
```

Comment:
Looking at this in context - it is the exact same path as `submission-details`. Do we really want two methods that do the exact same thing? The path is the same whether you are using a `GET` or a `DELETE` ... why do we need two code paths for the single case?